[2024-12-17T08:38:02.673+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: players_data_to_postgres_etl.load_data manual__2024-12-17T08:37:56.869716+00:00 [queued]>
[2024-12-17T08:38:02.677+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: players_data_to_postgres_etl.load_data manual__2024-12-17T08:37:56.869716+00:00 [queued]>
[2024-12-17T08:38:02.677+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-12-17T08:38:02.682+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): load_data> on 2024-12-17 08:37:56.869716+00:00
[2024-12-17T08:38:02.686+0000] {standard_task_runner.py:57} INFO - Started process 11632 to run task
[2024-12-17T08:38:02.688+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'players_data_to_postgres_etl', 'load_data', 'manual__2024-12-17T08:37:56.869716+00:00', '--job-id', '364', '--raw', '--subdir', 'DAGS_FOLDER/players_data_to_postgres_etl.py', '--cfg-path', '/tmp/tmpwi3gytgq']
[2024-12-17T08:38:02.689+0000] {standard_task_runner.py:85} INFO - Job 364: Subtask load_data
[2024-12-17T08:38:02.722+0000] {task_command.py:410} INFO - Running <TaskInstance: players_data_to_postgres_etl.load_data manual__2024-12-17T08:37:56.869716+00:00 [running]> on host 653f470dc9b4
[2024-12-17T08:38:02.786+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='players_data_to_postgres_etl' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-17T08:37:56.869716+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-17T08:37:56.869716+00:00'
[2024-12-17T08:38:02.815+0000] {logging_mixin.py:149} INFO - Data loaded to PostgreSQL in table 'players'
[2024-12-17T08:38:02.815+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-12-17T08:38:02.823+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=players_data_to_postgres_etl, task_id=load_data, execution_date=20241217T083756, start_date=20241217T083802, end_date=20241217T083802
[2024-12-17T08:38:02.868+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-12-17T08:38:02.880+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
